app/components/DepositWithdraw/DepositWithdrawAssetSelector.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { connect } from "alt-react";
+import Translate from "react-translate-component";
+import BindToChainState from "../Utility/BindToChainState";
+import { Apis } from "bitsharesjs-ws";
+import { settingsAPIs, rudexAPIs } from "api/apiConfig";
+import GatewayStore from "stores/GatewayStore";
+import GatewayActions from "actions/GatewayActions";
+import TypeAhead from "../Utility/TypeAhead";
+
+class DepositWithdrawAssetSelector  extends React.Component {
+    render(){
+        const { props } = this;
+
+        let getCoinOption = (item)=>{
+            return {id: item.symbol, label: item.symbol}
+        }
+
+        let coinItems = props.openLedgerBackedCoins.map(getCoinOption).concat(props.rudexBackedCoins.map(getCoinOption)).concat(props.blockTradesBackedCoins.map(getCoinOption));
+
+        return <TypeAhead items={coinItems} />
```

Comment:
Include props so we can pass back the value to the initiating page where the form is

`return <TypeAhead items={coinItems} {...this.props} />;`